packages/itwin/measure-tools/src/tools/MeasureAreaTool.ts

```diff
@@ -24,13 +26,17 @@ import { MeasurementViewTarget } from "../api/MeasurementViewTarget";
 import type { AreaMeasurement } from "../measurements/AreaMeasurement";
 import { MeasureAreaToolModel } from "../toolmodels/MeasureAreaToolModel";
 import { MeasureTools } from "../MeasureTools";
+import { SheetMeasurementsHelper } from "../api/SheetMeasurementHelper";
+import type { DrawingMetaData } from "../api/Measurement";
 
 export class MeasureAreaTool extends MeasurementToolBase<
 AreaMeasurement,
 MeasureAreaToolModel
 > {
   public static override toolId = "MeasureTools.MeasureArea";
   public static override iconSpec = "icon-measure-2d";
+  private _enableSheetMeasurements;
```

Comment:
This variable is marked as 'any' by default if you don't provide proper type information or default assignment. Same in MeasureDistanceTool.